src/main/java/Model/Party.java

```diff
@@ -0,0 +1,92 @@
+package Model;
+
+public class Party {
+    private  int    partyId;
+    private String firstName;
+    private String  lastname;
```

Comment:
lastname should be written as lastName. Please follow the same naming convention throughout the application.